src/plugins/index_pattern_field_editor/public/components/field_editor_flyout_content.tsx

```diff
@@ -53,20 +53,29 @@ const geti18nTexts = (field?: Field) => {
     confirmButtonText: i18n.translate(
       'indexPatternFieldEditor.deleteRuntimeField.confirmationModal.saveButtonLabel',
       {
-        defaultMessage: 'Save',
+        defaultMessage: 'Save changes',
       }
     ),
     warningChangingFields: i18n.translate(
       'indexPatternFieldEditor.deleteRuntimeField.confirmModal.warningChangingFields',
       {
         defaultMessage:
-          'Warning: Changing name or type may break searches or visualizations that rely on this field.',
+          'Changing name or type can break searches and visualizations that rely on this field.',
       }
     ),
     typeConfirm: i18n.translate(
       'indexPatternFieldEditor.saveRuntimeField.confirmModal.typeConfirm',
       {
-        defaultMessage: "Type 'CHANGE' to continue:",
+        defaultMessage: 'Enter CHANGE to continue:',
```

Comment:
Here we have trailing `:`, but in `Enter REMOVE to confirm` we don't 
Not sure which one is expected